scheduler/test/cook/test/mesos/fenzo_utils.clj

```diff
@@ -85,3 +89,16 @@
                         "mem" 2
                         "ports" 2
                         "disk" 1}}))))
+
+
+(deftest test-record-placement-failures
+  (let [uri "datomic:mem://test-record-placement-failures"
+        conn (restore-fresh-database! uri)
+        job-id (create-dummy-job conn :under-investigation true)
+        job (->> job-id
+                 (d/entity (d/db conn))
+                 util/entity->map)
+        ^TaskRequest task-request (scheduler/make-task-request job)
+        failure (assignment-failure :mem)
+        assignment-result (SimpleAssignmentResult. [failure] nil task-request)]
+    (is (fenzo/record-placement-failures! conn [[assignment-result]]))))
```

Comment:
Is this just asserting that transacting the placement failures didn't throw? Should we also assert on what got transacted?

Reply:
This is just asserting that it returned non-nil (a transaction was performed). I will add an additional query afterwards to check the results.